### ISSUE TITLE
return type do not suuport state_is_tuple

### ISSUE THREAD

**Issue by lfzCarlosC** [tam17aki/recurrent-batchnorm-tensorflow#2]
should  change return type to 

new_state = (LSTMStateTuple(c, h) if self._state_is_tuple else
                 array_ops.concat([c, h], 1))

**Comment by lfzCarlosC** [comment]
return h, new_state

**Comment by tam17aki** [comment]
Hi,

Thank you for report. 
I'll fix it.

Best regards,
Akira
